trimmer_decorator.rb.rb

```diff
@@ -0,0 +1,6 @@
+class TrimmerDecorator < Decorator
```

Comment:
- [ ] Kindly import the `Decorator` class before inheriting it to avoid any error.